doc/source/oss-users.rst

```diff
@@ -0,0 +1,33 @@
+Open source projects using Ray
+==============================
+
+The Ray ecosystem is rapidly growing! This page lists projects and companies that are known to use Ray. This does not constitute an endorsement.
```

Comment:
Let's just include open source projects. It's not useful to users to know what companies are using ray, they can see ray.io for that.